test/cpp_visitor.cpp

```diff
@@ -0,0 +1,74 @@
+#include <cppast/cpp_entity.hpp>
```

Comment:
It's visitor.hpp, not cpp_visitor.hpp, so visitor.cpp, instead of cpp_visitor.cpp